packages/common-all/src/fuse.ts

```diff
@@ -63,7 +63,7 @@ export const FuseExtendedSearchConstants = {
 //         });
 //       const path = `${dir}/${THRESHOLD_VALUE}_${new Date().getTime()}.json`;
 //       require('fs').writeFile(path, data, ()=>{});
-const THRESHOLD_VALUE = 0.2;
+export const DEFAULT_THRESHOLD_VALUE = 0.2;
```

Comment:
instead of having a hardcoded value here, can you add it as a config value via [[Add New Config|dendron://dendron.docs/pkg.common-all.dev.cook#add-new-config]]?

this should also fix the test failing

Reply:
+1 on @kevinslin's suggestion. I think removing the constant and having `fuzzThreshold` config required and default to `0.2` would be better.

Reply:
Added default config value